### ISSUE TITLE
Updating from 5.5.0 to 5.5.1 crashed after installation

### ISSUE THREAD

**Issue by dfg555** [Belphemur/SoundSwitch#495]
**What were you doing when it crashed ?**
Doing an update by clicking the blinking system tray icon of SoundSwitch. However it seems like the update was able to complete successfully as I have v5.5.1.30746 installed.

Event Viewer:
Faulting application name: SoundSwitch.exe, version: 5.5.1.30746, time stamp: 0x5ee99cfc
Faulting module name: KERNELBASE.dll, version: 10.0.18362.959, time stamp: 0x7b90c1b5
Exception code: 0xe0434352
Fault offset: 0x000000000003a799
Faulting process id: 0x2e98
Faulting application start time: 0x01d676d57793d0ec
Faulting application path: C:\Program Files\SoundSwitch\SoundSwitch.exe
Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
Report Id: de2c7c83-ce38-4f2e-ab69-e8fe2e4de001
Faulting package full name: 
Faulting package-relative application ID: 

**ZIP**
Drag & Drop the file
[SoundSwitch-crashlog-20_8_2020.zip](https://github.com/Belphemur/SoundSwitch/files/5102143/SoundSwitch-crashlog-20_8_2020.zip)

**Comment by Darktortue** [comment]
I've got the same problem here.
A 5.5.1 fresh install is working correctly but the update from 5.5.0 to 5.5.1 isn't working. The software crashed and then we can't use it anymore.

My logs : 
[SoundSwitch-crashlog-20_8_2020.zip](https://github.com/Belphemur/SoundSwitch/files/5102481/SoundSwitch-crashlog-20_8_2020.zip)

**Comment by qbah91** [comment]
Hello,

The same issue here.

My logs:
[SoundSwitch-crashlog-20_8_2020.zip](https://github.com/Belphemur/SoundSwitch/files/5102967/SoundSwitch-crashlog-20_8_2020.zip)

**Comment by 4girl1der** [comment]
Yep, This happened to me to. Had to reboot to get the system stable.

**Comment by millerlarson** [comment]
Had the issue upon upgrade.  Tried a fresh install of 5.5.1, same issue.  1250ms timeout error, hotkey wasn't working, etc.  Reboot seemed to resolve.
